Automated Smoketest: :x:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 17baf9c1837f99d5bfda4565c9db39d6605efc89
Build: success
ECL Watch:  Rebuilding Site
Lint error(s): 
npm ERR! Test failed.  See above for more details.

Install hpccsystems-platform-community_7.0.0-rc1.el7.x86_64.rpm
Creating group hpcc ...\x1b[32m [   OK    ]  \x1b[0m 
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |89 |89 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |805 |805 |0 |
|test (thor) |729 |729 |0 |
|test (roxie) |878 |878 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

[Build error:](https://gist.github.com/ff99bd9ab30f97f7026b306058285eac)
Build log [Build log](https://gist.github.com/HPCCSmoketest/ff99bd9ab30f97f7026b306058285eac#file-result-18-08-17-10-49-19-log)

Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   31 sec (00:00:31) |  177 sec (00:02:57) |  147 sec (00:02:27) |    8 sec (00:00:08) |   29 sec (00:00:29) | 1335 sec (00:22:15) |   17 sec (00:00:17) | 1744 sec (00:29:04) |

--- a/esp/src/eclwatch/DynamicESDLMethodWidget.js
+++ b/esp/src/eclwatch/DynamicESDLMethodWidget.js
@@ -167,8 +167,7 @@ define([
                             __hpcc_parentName: null,
                             __hpcc_id: row.Name
                         });
-
-                        if (row.Attributes) {
+                        if (row.XML) {
                             newRows.push({
                                 __hpcc_parentName: row.Name,
                                 __hpcc_id: row.Name + idx,

